feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html

```diff
@@ -18,12 +18,15 @@
         class="cx-asm-customer-list"
         *ngIf="customerSupportAgentLoggedIn$ | async"
       >
-        <cx-icon [type]="iconTypes.USER_FRIENDS"></cx-icon>
         <a
           #customerListLink
+          tabindex="0"
+          role="button"
           class="cx-asm-customer-list-link"
           (click)="showCustomList()"
-          ><span>{{ 'asm.customers' | cxTranslate }}</span></a
+        >
+          <cx-icon [type]="iconTypes.USER_FRIENDS"></cx-icon>
+          <span>{{ 'asm.customers' | cxTranslate }}</span></a
```

Comment:
To double-check - do we need an aria-label here?

Reply:
I don't think so because usually, we use aria-label for non-text links and buttons... also AMP doesn't complain.